tomatoTomato/ContentView.swift

```diff
@@ -8,11 +8,41 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    @State private var isPresentingNewTaskView = false
+    
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        NavigationView {
+            Text("There will be a list of tasks here later")
+
+            .navigationBarTitle("To do:")
+            
+            .toolbar {
+                Button(action: {
+                    isPresentingNewTaskView = true
+                }) {
+                    Image(systemName: "plus").imageScale(.large)
+                }
+                .accessibilityLabel("New Task")
+            }
+            .sheet(isPresented: $isPresentingNewTaskView) {
+                NavigationView {
```

Comment:
please move this NavigationView to a variable. This way this long list will be more readable and more maintainable in the future

Reply:
cannot do because NewTaskView is already a separate entity 
everything else here is part of the structure